tensorrt_llm/_torch/attention_backend/sparse/minimax_m3/metadata.py

```
@@ -0,0 +1,892 @@
+            # SGLang exposes only "max" today and that is what the MiniMax-M3
+            # checkpoint config specifies. Reject anything else explicitly so
+            # a config drift surfaces immediately.
+            raise ValueError(
+                f"score_type={self.score_type!r} is not supported "
+                "(only 'max' matches the SGLang reference)"
+            )
+
+    @classmethod
+    def from_sparse_attention_config(
+        cls,
+        sparse_attention_config: "MiniMaxM3SparseAttentionConfig",
```

Flake8 (7.3.0)

[error] undefined name 'MiniMaxM3SparseAttentionConfig'

(F821)